index.html

```diff
@@ -135,10 +135,66 @@ <h1>Multi-Post <br> Stories</h1>
           <li class="ruby_tag">Ruby on rails</li>
           <li>javaScript</li>
         </ul>
-        <button>See Project</button>
+        <button onclick="showPopup(3)">See Project</button>
       </div>
     </div>
   </section>
+  <div id="popup">
+		<div id="popup-content">
+			<h2 id="popup-title"></h2>
+      <span id="close-popup" onclick="hidePopup()">&times;</span>
+      <p id="popup-summary"></p>
+      <img id="popup-image"/>
+      <div id="popup-details">
+        <div id="popup-description">
+          <div id="popup-description1"></div>
+          <p id="popup-description2"></p>
+        </div>
+        <div id="popup_tag-buttons">
+          <ul id="popup-technologies">
+            <li id="popup-tag1"></li>
+            <li id="popup-tag2"></li>
+            <li id="popup-tag3"></li>
+          </ul>
+          <ul class="popup-technologies_2">
+            <li id="popup-tag4"></li>
+            <li id="popup-tag5"></li>
+          </ul>
+          <hr/>
+          <button class="popup-btn" id="popup-btn1" onclick="liveLink()">See live <img src="images/live.png"></button>
```

Comment:
- [ ] Please kindly ensure that the popup of each project is generated dynamically using Javascript when a user clicks on the `See project` button. Please remove all HTML content on the popup and generate with Javascript.